TELEMETRY.md

```diff
@@ -9557,40 +9557,13 @@ No properties for event
 ## Description
 
 
-
- Telemetry event sent to provide information on whether we have successfully identify the type of shell used.
- This information is useful in determining how well we identify shells on users machines.
- This impacts extraction of env variables from current shell.
- So, the better this works, the better it is for the user.
- failed - If true, indicates we have failed to identify the shell. Note this impacts impacts ability to activate environments in the terminal & code.
- shellIdentificationSource - How was the shell identified. One of 'terminalName' | 'settings' | 'environment' | 'default'
-                             If terminalName, then this means we identified the type of the shell based on the name of the terminal.
-                             If settings, then this means we identified the type of the shell based on user settings in VS Code.
-                             If environment, then this means we identified the type of the shell based on their environment (env variables, etc).
-                                 I.e. their default OS Shell.
-                             If default, then we reverted to OS defaults (cmd on windows, and bash on the rest).
-                                 This is the worst case scenario.
-                                 I.e. we could not identify the shell at all.
- hasCustomShell - If undefined (not set), we didn't check.
-                  If true, user has customzied their shell in VSC Settings.
- hasShellInEnv - If undefined (not set), we didn't check.
-                 If true, user has a shell in their environment.
-                 If false, user does not have a shell in their environment.
```

Comment:
As you can see now all of the documentaiton is lost from the MD file as well.
I don' think this is desirable .